src/components/LoginForm/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
```

Comment:
this component is so similar to the register, for the future think about reusability and abstraction